aiosmtpd/smtp.py

```diff
@@ -3,10 +3,12 @@
 import logging
 import collections
 
+from .context import MessageContext, SessionContext
```

Comment:
I generally prefer absolute imports, e.g. `aiosmtpd.context` but of course if we boil this down, we might not need a separate module.  We could just add it to `smtp.py`.